### PR TITLE
Update and Add New Quests Related to Fusion Power Changes

### PR DESCRIPTION
**Pull request by Steelux8**
The new questline for the fusion chains that I added before is in the top right. Additionally, I added one quest for each XL Turbine, including information about mechanics, blacklists, penalty formulas and known bugs. Finally, I added some notes regarding the chains to the nuclear power tab.

All the quests I added have insignificant rewards, or none at all, given that they are all lategame/endgame. This applies to the earlier XL Turbine quests too, which also doesn't have rewards.

### CODE REVIEW

File: config/betterquesting/DefaultQuests/Quests/GT-AAAAAAAAAAAAAAAAAAAAEQ==/CelestialTungste-LSqzsH4oS7OQrZyolb7C0A==.json [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+      "snd_update:8": "random.levelup",
+      "repeatTime:3": -1,
+      "globalShare:1": 0,
+      "questLogic:8": "AND",
+      "repeat_relative:1": 1,
+      "name:8": "Celestial Tungsten Plasma - The Mk3 Finale",
+      "lockedProgress:1": 0,
+      "autoClaim:1": 0,
+      "isSilent:1": 0,
+      "desc:8": "This is the end of the mk3 fusion chain, and the three chains altogether. After 14 fusionr recipes on top of Helium Plasma, you hold in your tanks the elusive Celestial Tungsten Plasma, whose fuel value of 720M makes it immune to XL Plasma penalties all the way to spacetime!\n\nUpscaling plasma powergen up to endgame, and the Dyson Swarm, is most likely reliant on either this or Americium Plasma, which is 1-step but not as powerful on a per-reactor basis."
```
Comment (chochem) [inline config/betterquesting/DefaultQuests/Quests/GT-AAAAAAAAAAAAAAAAAAAAEQ==/CelestialTungste-LSqzsH4oS7OQrZyolb7C0A==.json:32]: typo fusionr -> fusion

File: config/betterquesting/DefaultQuests/Quests/GT-AAAAAAAAAAAAAAAAAAAAEQ==/AdvancedXeninol-D0JaazeIQ8qctaMx9xTqEw==.json [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+      "snd_update:8": "random.levelup",
+      "repeatTime:3": -1,
+      "globalShare:1": 0,
+      "questLogic:8": "AND",
+      "repeat_relative:1": 1,
+      "name:8": "Advanced Xeninol",
+      "lockedProgress:1": 0,
+      "autoClaim:1": 0,
+      "isSilent:1": 0,
+      "desc:8": "It\u0027s here that you might yourself with some useful stuff alongside power. Xenon will soon be useful to use in the EBF, and Advanced Nitinol? If you don\u0027t already have a Quantum Anomaly setup for it, you can use this. Who knows what\u0027ll come in the future?"
```
Comment (chochem) [inline config/betterquesting/DefaultQuests/Quests/GT-AAAAAAAAAAAAAAAAAAAAEQ==/AdvancedXeninol-D0JaazeIQ8qctaMx9xTqEw==.json:40]: will soon be -> is (ebf change is already merged)

File: config/betterquesting/DefaultQuests/Quests/Tier8UV-AAAAAAAAAAAAAAAAAAAALQ==/IPutFusioninYour-AAAAAAAAAAAAAAAAAAAEsg==.json [hunk 3]
```diff
@@ -29,7 +29,7 @@
       "lockedProgress:1": 0,
       "autoClaim:1": 0,
       "isSilent:1": 0,
-      "desc:8": "To make Oganesson for your Void Miners, you\u0027ll need to make Californium, to make that, you\u0027ll need Beryllium and Plutonium 239, all in the Reactor.\n\nSeriously consider replicating it after the first cell."
+      "desc:8": "To make Oganesson, you\u0027ll need to make Californium. To make that, you\u0027ll need Beryllium and Plutonium 239, all in the Fusion Reactor. After the two fusion chains, you\u0027ll have the best noble gas in the pack, which is used in several places.\n\nIn this tier, you can send it to Void Miners, to increase their ore production speed, or EBFs, to make them run faster, although the EBF option is too weak right now (changes coming soon). Later on, Oganesson will be used in other places, like the optical circuit line, or to produce Metastable Oganesson, which is needed in much larger quantities later on.\n\nSeriously consider replicating it after the first cell."
```
Comment (chochem) [inline config/betterquesting/DefaultQuests/Quests/Tier8UV-AAAAAAAAAAAAAAAAAAAALQ==/IPutFusioninYour-AAAAAAAAAAAAAAAAAAAEsg==.json:32]: are we still recommending replication? how do the numbers look with your change. are we not planning to remove that?

File: config/betterquesting/DefaultQuests/Quests/GT-AAAAAAAAAAAAAAAAAAAAEQ==/AdvancedXeninol-D0JaazeIQ8qctaMx9xTqEw==.json (hunk 2 above)
Comment (Steelux8) [reply config/betterquesting/DefaultQuests/Quests/GT-AAAAAAAAAAAAAAAAAAAAEQ==/AdvancedXeninol-D0JaazeIQ8qctaMx9xTqEw==.json]: Yeah, I was working on this before the merge happened, thanks for letting me know.

File: config/betterquesting/DefaultQuests/Quests/Tier8UV-AAAAAAAAAAAAAAAAAAAALQ==/IPutFusioninYour-AAAAAAAAAAAAAAAAAAAEsg==.json (hunk 3 above)
Comment (Steelux8) [reply config/betterquesting/DefaultQuests/Quests/Tier8UV-AAAAAAAAAAAAAAAAAAAALQ==/IPutFusioninYour-AAAAAAAAAAAAAAAAAAAEsg==.json:32]: I'll change the text to instead reflect that replication is on the way out, and the fusion recipe is better now.

**Review by chochem — APPROVED** [review]
thanks!

**Pull request merged**